internal/backend/crypto/gpg/cli/binary_windows.go

```diff
@@ -3,14 +3,44 @@
 package cli
 
 import (
+	"context"
+	"errors"
 	"os/exec"
 	"path/filepath"
+	"sort"
 
 	"golang.org/x/sys/windows/registry"
 
+	"github.com/gopasspw/gopass/internal/debug"
 	"github.com/gopasspw/gopass/pkg/fsutil"
 )
 
+func detectBinary(bin string) (string, error) {
```

Comment:
is this using the `GOPASS_GPG_BINARY` when it's set?

Reply:
Yes, through the loader.